discovery-client/build.gradle

```diff
@@ -18,24 +18,28 @@ dependencies {
     // used by AWSParameterStoreConfigClient
     compileOnly group: 'com.amazonaws', name: 'aws-java-sdk-ssm', version: '1.11.683'
 
-    testCompile "org.testcontainers:spock:1.10.1"
+    testImplementation "org.testcontainers:spock:1.10.1"
 
-    testCompile dependencyVersion("micronaut.aws"), {
+    testImplementation dependencyVersion("micronaut.aws"), {
         exclude module:'micronaut-http-client'
         exclude module:'micronaut-inject'
     }
-    testCompile group: 'com.amazonaws', name: 'aws-java-sdk-route53', version: '1.11.297'
-    testCompile group: 'com.amazonaws', name: 'aws-java-sdk-core', version: '1.11.689'
-    testCompile group: 'com.amazonaws', name: 'jmespath-java', version: '1.11.297'
-    testCompile group: 'com.amazonaws', name: 'aws-java-sdk-servicediscovery', version: '1.11.297'
-    testCompile group: 'com.amazonaws', name: 'aws-java-sdk-ssm', version: '1.11.308'
 
-    testCompile project(":management")
-    testCompile project(":http-server-netty")
-    testCompile project(":inject-java")
-    testCompile project(":inject-groovy")
-    testCompile "com.amazonaws:aws-java-sdk-ec2:1.11.683"
-    testCompile dependencyVersion("reactor")
-    testRuntime dependencyVersion("snakeyaml")
+    testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-route53', version: '1.11.297'
+    testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-core', version: '1.11.689'
+    testImplementation group: 'com.amazonaws', name: 'jmespath-java', version: '1.11.297'
+    testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-servicediscovery', version: '1.11.297'
+    testImplementation group: 'com.amazonaws', name: 'aws-java-sdk-ssm', version: '1.11.308'
 
+    testImplementation project(":management")
+    testImplementation project(":http-server-netty")
+    testImplementation project(":inject-java")
+    testImplementation project(":inject-groovy")
+    testImplementation "com.amazonaws:aws-java-sdk-ec2:1.11.683"
+    testImplementation dependencyVersion("reactor")
+    testAnnotationProcessor project(":management")
+    testAnnotationProcessor project(":http-server-netty")
```

Comment:
Why do management and http-server-netty need to be on the annotation processor classpath?

Reply:
Good question